Resources/Textures/_Harmony/Clothing/Head/Hats/hopcap.rsi/meta.json

```
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from the tg station atcommithttps://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e and then resprited by TadJohnson00 and Timemaster99 for DeltaV atcommithttps://github.com/DeltaV-Station/Delta-v/pull/1501, and then resprited a third time by CrimsonJupiter for Harmony",
```

* Multiple spaces missing in copyright.
```suggestion
  "copyright": "Taken from the tg station at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e and then resprited by TadJohnson00 and Timemaster99 for DeltaV at commit https://github.com/DeltaV-Station/Delta-v/pull/1501, and then resprited a third time by CrimsonJupiter for Harmony",
```

* `at commit https://github.com/DeltaV-Station/Delta-v/pull/1501` -- PR request is not a commit. Find a commit it was merged to master in.